components/pd_client/src/client.rs

```diff
@@ -146,18 +151,30 @@ impl RpcClient {
 
     /// Gets the leader of PD.
     pub fn get_leader(&self) -> Member {
-        self.leader_client.get_leader()
+        self.pd_client.get_leader()
+    }
+
+    /// Gets the forwarded host
+    pub fn get_forwarded_host(&self) -> String {
+        self.pd_client.get_forwarded_host()
     }
 
     /// Re-establishes connection with PD leader in synchronized fashion.
     pub fn reconnect(&self) -> Result<()> {
-        block_on(self.leader_client.reconnect(true))
+        block_on(self.pd_client.reconnect(true))
     }
 
     /// Creates a new call option with default request timeout.
     #[inline]
-    fn call_option() -> CallOption {
-        CallOption::default().timeout(Duration::from_secs(REQUEST_TIMEOUT))
+    fn call_option(forwarded_host: &str) -> CallOption {
```

Comment:
Why not define it as `fn call_option(client: &Client) -> CallOption`? So `get_forwarded_host` can be removed.